protocol.html

```diff
@@ -1168,6 +1169,47 @@ <h3 property="schema:name">Security and Privacy Review</h3>
+
+                    <dt about="#societal-impact-review-what-effect-may-features-of-your-specification-have-on-minority-groups" id="societal-impact-review-what-effect-may-features-of-your-specification-have-on-minority-groups"><a href="https://w3ctag.github.io/societal-impact-questionnaire/#what-effect-may-features-of-your-specification-have-on-minority-groups" rel="cito:repliesTo">What effect may features of your specification have on minority groups?</a></dt>
+                    <dd about="#societal-impact-review-what-effect-may-features-of-your-specification-have-on-minority-groups" datatype="rdf:HTML" property="schema:description">..</dd>
+
+                    <dt about="#societal-impact-review-what-are-the-power-dynamics-at-play-in-implementations-of-your-specification" id="societal-impact-review-what-are-the-power-dynamics-at-play-in-implementations-of-your-specification"><a href="https://w3ctag.github.io/societal-impact-questionnaire/#what-are-the-power-dynamics-at-play-in-implementations-of-your-specification" rel="cito:repliesTo">What are the power dynamics at play in implementations of your specification?</a></dt>
+                    <dd about="#societal-impact-review-what-are-the-power-dynamics-at-play-in-implementations-of-your-specification" datatype="rdf:HTML" property="schema:description">..</dd>
+
+                    <dt about="#societal-impact-review-what-points-of-centralization-does-your-feature-bring-to-the-web-platform" id="societal-impact-review-what-points-of-centralization-does-your-feature-bring-to-the-web-platform"><a href="https://w3ctag.github.io/societal-impact-questionnaire/#what-points-of-centralization-does-your-feature-bring-to-the-web-platform" rel="cito:repliesTo">What points of centralization does your feature bring to the web platform?</a></dt>
+                    <dd about="#societal-impact-review-what-points-of-centralization-does-your-feature-bring-to-the-web-platform" datatype="rdf:HTML" property="schema:description">..</dd>
+
+                    <dt about="#societal-impact-review-to-what-extent-do-the-features-in-your-specification-result-in-increased-power-consumption-or-emissions" id="societal-impact-review-to-what-extent-do-the-features-in-your-specification-result-in-increased-power-consumption-or-emissions"><a href="https://w3ctag.github.io/societal-impact-questionnaire/#to-what-extent-do-the-features-in-your-specification-result-in-increased-power-consumption-or-emissions" rel="cito:repliesTo">To what extent do the features in your specification result in increased power consumption or emissions?</a></dt>
+                    <dd about="#societal-impact-review-to-what-extent-do-the-features-in-your-specification-result-in-increased-power-consumption-or-emissions" datatype="rdf:HTML" property="schema:description">..</dd>
```

Comment:
@csarven -- This reads as `To what extent do the features in your specification result in increased power consumption or [unspecified] emissions?`  The most obvious, `power emissions`, would seem to be a positive.  Emissions of H2O or O2 would also likely be fine, so it's not `any emissions`.  Therefore, assuming greenhouse gases are the actual concern, this should change from `power consumption or emissions` to **`power consumption or greenhouse gas emissions`**.